src/gluonts/evaluation/_base.py

```diff
@@ -399,13 +399,13 @@ def get_metrics_per_ts(
             "MSE": mse(pred_target, mean_fcst)
             if mean_fcst is not None
             else None,
-            "abs_error": abs_error(pred_target, median_fcst),
+            "abs_error": abs_error(pred_target, mean_fcst),
             "abs_target_sum": abs_target_sum(pred_target),
             "abs_target_mean": abs_target_mean(pred_target),
             "seasonal_error": seasonal_error,
-            "MASE": mase(pred_target, median_fcst, seasonal_error),
-            "MAPE": mape(pred_target, median_fcst),
-            "sMAPE": smape(pred_target, median_fcst),
+            "MASE": mase(pred_target, mean_fcst, seasonal_error),
+            "MAPE": mape(pred_target, mean_fcst),
+            "sMAPE": smape(pred_target, mean_fcst),
```

Comment:
I think we can split changes into multiple pull requests. Maybe you can keep this one about adding the option for Fourier features?